src/scratchtocatrobat/converter/converter.py

```diff
@@ -1423,3 +1429,34 @@ def _convert_answer_block(self):
                                                                                           self.script_context)
         return _variable_for(shared_global_answer_user_variable.getName())
 
+    @_register_handler(_block_name_to_handler_map, "createCloneOf")
+    def _convert_create_clone_of_block(self):
+        [base_sprite] = self.arguments
+        if isinstance(base_sprite, catformula.FormulaElement):
+            return catbricks.NoteBrick("Can't convert Clone-Brick with Formula as argument.\n")
+
+        if len(base_sprite) == 0:
+            return catbricks.NoteBrick("Can't convert Clone-Brick with no argument.\n")
+
+        if isinstance(base_sprite, basestring):
```

Comment:
Good solution! 👍 
Better than checking for both str and unicode types separately.